### PR TITLE
Added onClick fn for pixel

### PR DESCRIPTION
**Pull request by thsubaku9**
## Checklist

- [x] I ran `npm test` locally and it passed without errors.

- [ ] I only edited the `_data/pixels.json` file.

- [ ] I entered the `username` in the `pixels.json` that I'm also using to create this pull request.
- [x] I acknowledge that all my contributions will be made under the project's [license](../LICENSE).

## Description

Adds the onClick functionality which redirects the user to that respective pixel owner's GitHub profile.

### CODE REVIEW

File: assets/javascript/party-mode.js [hunk 1]
```diff
@@ -203,3 +203,8 @@ function allCells() {
   }
   return all;
 }
+
+function gotoPage(username){
```
Comment (dkundel) [inline assets/javascript/party-mode.js]: Can we move this function to `name-hover.js` instead? Seems closer related to that file.

File: index.njk [hunk 2]
```diff
@@ -82,6 +82,7 @@
                 name="{% if pixel.username %}{{pixel.username}} {% endif %}"
                 data-pixel-x="{% if pixel.x %}{{pixel.x}} {% endif %}"
                 data-pixel-y="{% if pixel.x %}{{pixel.y}} {% endif %}"
+                onClick = "{% if pixel.username%} gotoPage( '{{pixel.username}}' ) {% endif %}"
```
Comment (dkundel) [inline index.njk]: Can we attach the event listener via JavaScript instead? Just to be consistent? You can add it to the `window.onload` event in `name-hover.js`

**Comment by thsubaku9** [comment]
Sorry for the late reply. I've shifted the function to name-hover.js; I haven't added the eventListener via JavaScript because it'll end making the file harder to read (compared to the simple Nunjucks utilization). I'll add it if it's important. Thanks in advance.

File: index.njk (hunk 2 above)
Comment (thsubaku9) [reply index.njk]: I'll need to add it individually for each element (while handling and obtaining the necessary information using only the event object) hence leading to a much more verbose and messy implementation ( as opposed to the cleaner current implementation). Hence I've left it as-is for easier future maintainability/modification

File: assets/javascript/party-mode.js (hunk 1 above)
Comment (thsubaku9) [reply assets/javascript/party-mode.js]: Shifted as per requirement

**Pull request closed without merging**